server/ERNI.PBA.Server.Host/Controllers/UserController.cs

```diff
@@ -53,7 +53,7 @@ public async Task<IActionResult> RegisterUser(CancellationToken cancellationToke
 
             await _unitOfWork.SaveChanges(cancellationToken);
 
-            return Ok();
+            return Ok(user);
```

Comment:
returning the user db entity is a bad idea, you never know what sensitive information it could contain